README.md

```diff
@@ -1,116 +1,76 @@
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/netmiko.svg)
 [![PyPI](https://img.shields.io/pypi/v/netmiko.svg)](https://pypi.python.org/pypi/netmiko)
 [![Downloads](https://pepy.tech/badge/netmiko)](https://pepy.tech/project/netmiko)
+![GitHub contributors](https://img.shields.io/github/contributors/ktbyers/netmiko.svg)
 [![Code Style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
 
 
 Netmiko
 =======
 
-Multi-vendor library to simplify Paramiko SSH connections to network devices  
+Multi-vendor library to simplify Paramiko SSH connections to network devices
```

Comment:
I think the line wraps/white-spacing start getting messed up in Markdown if those two spaces at the end of the line go away.

Reply:
This might be wrong or not relevant...so you can probably ignore it...